### PR TITLE
[Content] [Bug Fix] Fuel Cans

### PR DESCRIPTION
**Pull request by Kurfursten**
- New fuel can; partitioned reagent container that easily swaps between two halves to allow you easy refills of a soldering iron, welding torch, or fire extinguisher depending on which reagent you want. It fits on all tool belts.
  fuel can
- Bug fix: CE belt can now hold CMCs, like the normal belt
- Bug fix: Open containers will no longer be stuffed into extinguishers. This lets you refill extinguishers without sticking the beaker inside.
- Tweak: Normal (and CE) toolbelts can now hold light replacers
- Tweak: CE belt can now hold station blueprints

### CODE REVIEW

**Comment by IratePirate** [comment]
I like this a lot.

**Comment by d3athrow** [comment]
Atomic PRs

**Comment by Kurfursten** [comment]
@d3athrow I'm sorry, I can't help myself

I'm getting better about it though

**Comment by d3athrow** [comment]
bbbbbbbbbbbbbbbbbbb

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -670,3 +671,51 @@
...
+	m_amt = 500
+	volume = 50
+	possible_transfer_amounts = list(5,10,20)
+	var/slot = 0 //This dictates which side is open
+	var/datum/reagents/slotzero = null
+	var/datum/reagents/slotone = null
+
+/obj/item/weapon/reagent_containers/glass/fuelcan/New()
+	..()
+	slotzero = new/datum/reagents(volume)
```
Comment (ComicIronic) [inline code/game/objects/items/weapons/tools.dm]: There is no need to make a new one.
Simply do slotzero = reagents.

**Comment by Kurfursten** [comment]
Please tag NFD

**Pull request merged**